docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md

```diff
@@ -162,7 +162,7 @@ var remote = require('selenium-webdriver/remote');
 driver.setFileDetector(new remote.FileDetector);  
   {{< / code-panel >}}
   {{< code-panel language="kotlin" >}}
-driver.setFileDetector(new LocalFileDetector())
+driver.fileDetector = LocalFileDetector()
```

Comment:
hi @marcus-rp,

Good catch.
Can you please update code for other translated pages too

Reply:
how do I do it?

Reply:
In the same directory you can find more files with the same name but a different suffixes... Those are the translations. Please also change the code in those files.
Don’t forget to sign the CLA please.